conf/mendel.config

```diff
@@ -2,21 +2,23 @@
 params {
   config_profile_name = 'MENDEL'
   config_profile_description = 'GMI MENDEL cluster profile provided by nf-core/configs'
-  config_profile_contact = 'Philipp H (@phue)'
+  config_profile_contact = 'Patrick Hüther (@phue)'
   config_profile_url = 'http://www.gmi.oeaw.ac.at/'
 }
 
-singularity {
-  enabled = true
+manifest {
```

Comment:
Is this functional code? I'm not so sure about this unfortunately...

Reply:
Surprisingly yes, it seems to work. It overrides the required nextflow version specified in the pipeline, which is needed here because the pbspro scheduler was only recently added.